src/cocaine/burlak/burlak.py

```diff
@@ -847,6 +851,12 @@ def process_loop(self):
         control_filter = yield self.get_filter_once()
         no_state_yet = True
 
+        try:
+            self.error('initializing feedback record')
+            yield self._init_state_record()
```

Comment:
Why can't `yield self.poster.post_state(self.ci_state)` be directly called here?